### ISSUE TITLE
metadata for ADO_CORINE_100m_3035

### ISSUE THREAD

**Issue by m-mohr** [Open-EO/openeo-spring-driver#35]
Hey,

I just had a quick look at your ADO_CORINE_100m_3035 collection while working on a web editor issue.

Some hints for improvements.
- The metadata should not do those "No <...> Information Available" summaries. Instead, just remove them from the output.
- The role "Producer" should be lower-cased.
- If you provide an e-mail in a "url" field, add "mailto:" at the beginning.
- eo:gsd doesn't exist anymore, it's now just "gsd"
- citation doesn't exist, I think you meant to use "sci:citation"? In this case also add "scientific" to "stac_extensions".

**Comment by clausmichele** [comment]
Hi @m-mohr, is there a "validation" tool for the STAC metadata? Maybe I've missed it.

**Comment by m-mohr** [comment]
Yes, there's stac-node-validator (JS, for stac >= 1.0.0-rc.1) and stac-validator (Python, I think it works for STAC 0.9, too), but due to the extensibility of JSON/STAC errors such as gsd instead of eo:gsd will not be spotted, but it likely shows you some of the issues mentioned above.
